lib/Onnxifi/GlowOnnxManager.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "GlowOnnxManager.h"
+
+#include "Base.h"
+
+namespace glow {
+namespace onnxifi {
+// static
```

Comment:
not sure if comment is useful :)

Reply:
Ok, I find these useful when reading code but if it's not in the style glow uses I'll remove it.